pkgs/development/tools/parsing/tree-sitter/update_impl.py

```diff
@@ -125,12 +154,19 @@ def fetchRepo() -> None:
                     version_rev=release
                 )
             )
+
+            # Load the data since we need the grammar path to execute the tests
+            data = json.loads(res)
+
+            # Test the grammar with Tree-sitter and set the isBroken flag accordingly
+            data['isBroken'] = not check_grammar(data)
```

Comment:
We should probably only add that if it applies, and remove it if not. This PR now changes lots of files for no good reason.